core/src/androidTest/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
                 android:name=".activities.RadioGroupActivity"/>
         <activity
                 android:theme="@style/Theme.AppCompat.Light"
-                android:name=".activities.AnimationActivity"/>
+                android:name=".activities.FrameActivity" />
```

Comment:
You should not remove AnimationActivity, right? By the contents of this PR FrameActivity is a new one.

Reply:
This activity doesn't exists anymore, should i remove it on a different PR?

Reply:
No, no no... you are right! THanks.